ci/test.yml

```diff
@@ -319,3 +319,25 @@ storage test:
       junit: tests/junit.xml
     expire_in: 1 week
     when: always
+
+# Hardware
+hardware legacy device test:
+  stage: test
+  extends: .legacy_job
+  tags:
+    - tpmb
+  dependencies:
+    - legacy fw debug build
+  script:
+    - cd ci/hardware_tests
+    - nix-shell --run "./record_video.sh ${CI_COMMIT_SHORT_SHA} start"
+    - nix-shell --run "cd ../.. && pipenv sync"
+    - nix-shell --run "pipenv run python bootstrap.py 1.8.3"
```

Comment:
I have remembered I wanted to comment this. Here I am using 1.8.3 but we should use the latest firmware. That could be done by either introducing something like `--latest` to trezorctl (I think users could benefit from that as well) or fetch it from somewhere.

Another solution is to not upload anything, but wipe the device from bootloader. The former however also tests an upgrade from the latest firmware, which is also nice.

Reply:
you can just not specify the version and you'll get the latest

Reply:
Ha, I didn't know, cool! Done